htgr/htr-10/steady/htr-10-full.i

```diff
@@ -0,0 +1,277 @@
+# ==============================================================================
+# Model description
+# Application : Griffin
+# ------------------------------------------------------------------------------
+# Idaho Falls, INL, June 8th, 2022
+# Author(s): Javier Ortensi
+# ==============================================================================
+# - HTR Full Configuration GRIFFIN neutronics input
+# - MainApp
+# ==============================================================================
+# - The Model has been built based on [1-2].
+# ------------------------------------------------------------------------------
+# [1] Benchmark Analysis of the HTR-10 with the MAMMOTH Reactor Physics
+#     Application, J. Ortensi et al.
+# [2] Evaluation of high temperature gas cooled reactor performance: benchmark
+#     analysis related to initial testing of the httr and htr-10. 
+#     Technical Report IAEA-TECDOC-1382, International Reactor Physics
+#     Experiment Evaluation Project, 2003.
+# ==============================================================================
+# MODEL PARAMETERS
+# ==============================================================================
+
+# State ------------------------------------------------------------------------
+# see state on xs library
+# Power ------------------------------------------------------------------------
+
+# ==============================================================================
+# GLOBAL PARAMETERS
+# ==============================================================================
+[GlobalParams]
+  coupled_flux_groups = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  scalar_flux = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  #Library options include xs/htr-10-full-1RI.xml, xs/htr-10-full-393K.xml, xs/htr-10-full-523K.xml,
+  # xs/htr-10-full-ARI.xml, and xs/htr-10-full-ARO.xml
```

Comment:
did you check all of these in?

Reply:
These all got merged into one master xs file.